File: questions/library/javascript/array.js
module.exports = {
  show: function (array) {
    let res = "[";
    for (let index in array) {
      if (index == 0) {
        res += array[index].toString();
      } else {
        res += ", " + array[index].toString();
      }
    }
    res += "]";
    return res;
  }
}

File: questions/library/go/helper/array.go
package helper

import "fmt"

func ArrayShow(array []int) (res string) {
	res = "["
	for index, entry := range array {
		if index == 0 {
			res += fmt.Sprintf("%d", entry)
		} else {
			res += fmt.Sprintf(", %d", entry)
		}
	}
	res += "]"
	return
}
